pkg/apiroutes/templates.go

```diff
@@ -54,8 +54,13 @@ type (
 
 // GetTemplates gets project templates from PFE's REST API.
 // Filter them using the function arguments
-func GetTemplates(projectStyle string, showEnabledOnly bool) ([]Template, error) {
-	req, err := http.NewRequest("GET", config.PFEOrigin()+"/api/v1/templates", nil)
+func GetTemplates(projectStyle string, showEnabledOnly bool, conID string) ([]Template, error) {
```

Comment:
to shorten this, you could group the input vars together for the strings like this.. 
`GetTemplates(projectStyle, conID string, showEnabledOnly bool)` 
This is a bit of a fussy comment but its good practice

Reply:
done

Reply:
personally preference I guess - I dont like that shorthand myself,  if we're doing it to shorten the number of args then it might be better to wrap them in a struct and pass that instead as one single arg.